packages/docs/getting-started/using-metamask-with-celo/README.md

```diff
@@ -0,0 +1,36 @@
+# Using MetaMask with Celo
+
+{% hint style="warning" %}
+MetaMask is primarily used for interacting with the Ethereum blockchain, and does not natively support Celo compatibility. As such, we recommend only sophisticated users and developers use it, at your own risk.
```

Comment:
Can we redirect back to the Celo Native Wallets here so users know where to go if they don't want to do all this?

"Please check out options for Celo native wallets [here]".